packages/kolibri-common/components/accordion/AccordionContainer.vue

```diff
@@ -0,0 +1,74 @@
+<template>
+
+  <div class="accordion">
+    <div
+      v-if="hasHeaderSlot"
+      class="header"
+      :style="{
+        borderColor: $themeTokens.fineLine,
+        backgroundColor: $themePalette.grey.v_50,
+        ...headerAppearanceOverrides,
+      }"
+    >
+      <slot
+        name="header"
+        :canExpandAll="canExpandAll"
+        :expandAll="expandAll"
+        :canCollapseAll="canCollapseAll"
+        :collapseAll="collapseAll"
+      ></slot>
+    </div>
+    <div>
+      <slot name="default"></slot>
+    </div>
+  </div>
+
+</template>
+
+
+<script>
+
+  import { computed } from 'kolibri.lib.vueCompositionApi';
+  import useAccordion from './useAccordion';
+
+  export default {
+    name: 'AccordionContainer',
+    setup(prop, { slots }) {
+      const { canExpandAll, canCollapseAll, collapseAll, expandAll } = useAccordion();
+
+      const hasHeaderSlot = computed(() => !!slots.header);
```

Comment:
For some reason it seems there is a bug when we use slots with slot-props and the vue Composition API, which is that outside the `setup` method, `this.$slots` is usually empty even if we do have slots.